sample-android-app/src/main/java/com/badoo/reaktive/sample/android/ReaktiveAcitivy.kt

```diff
@@ -0,0 +1,54 @@
+@file:Suppress("EXPERIMENTAL_API_USAGE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.badoo.reaktive.sample.android
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import com.badoo.reaktive.disposable.scope.DisposableScope
+import com.badoo.reaktive.disposable.scope.disposableScope
+import com.badoo.reaktive.disposable.scope.doOnDispose
+import com.badoo.reaktive.observable.observable
+import com.badoo.reaktive.observable.subscribeOn
+import com.badoo.reaktive.scheduler.computationScheduler
+import com.badoo.reaktive.scheduler.ioScheduler
+import com.badoo.reaktive.single.singleFromFunction
+import com.badoo.reaktive.single.subscribeOn
+
+    @SuppressLint("Registered")
+
+
+    class ReaktiveActivity : AppCompatActivity() {
```

Comment:
What is it?
Why it is in this PR?
What's up with formatting?

Reply:
By accident, removed